ui/shared/components/panel/variants/SavedVariants.jsx

```diff
@@ -74,17 +71,15 @@ class SavedVariants extends React.PureComponent {
 
   render() {
     const {
-      match, tableState, filters, discoveryFilters, totalPages, variantsToDisplay, totalVariantsCount, firstRecordIndex,
+      match, tableState, filters, totalPages, variantsToDisplay, totalVariantsCount, firstRecordIndex,
       tableSummaryComponent, loading, filteredVariantsCount, tagOptions, additionalFilter, updateTableField,
-      variantExportConfig, loadVariants, error,
+      variantExportConfig, loadVariants, error, multiple, getSelectedTag,
     } = this.props
     const { showAllFilters } = this.state
-    const { familyGuid, variantGuid, tag } = match.params
-
-    const appliedTagCategoryFilter = tag || (variantGuid ? null : (tableState.categoryFilter || ALL_FILTER))
+    const { familyGuid, variantGuid } = match.params
+    const optionValue = getSelectedTag(tableState.categoryFilter)
```

Comment:
notpick, but theres no need to assign this to a variable, it would be more readable if you directly assigned it to the prop